release-schema.json

```
@@ -24,6 +24,11 @@
                 "null"
               ],
               "format": "date-time"
+            },
+            "documentAvailabilityPeriod": {
+              "title": "Document availability period",
+              "description": "Deadline for the receipt of requests for documents or for accessing documents.",
```

Reworded to frame as period, instead of date/deadline, and to frame from single perspective of supplier (who accesses and submits) rather that both the supplier (accesses) and buyer (receives):

```suggestion
              "description": "The period during which documents can be accessed, or during which requests for access to documents can be submitted.",
```